examples/nft-nd-nns/nns_test.go

```diff
@@ -365,6 +365,7 @@ func TestTransfer(t *testing.T) {
 	cFrom.Invoke(t, 1, "totalSupply")
 	cFrom.Invoke(t, to.ScriptHash().BytesBE(), "ownerOf", "neo.com")
 
+	e.DisableCoverage() // contracts below have no source files which leads to unprocessable coverage data
```

Comment:
Coverage is not collected for example contracts. So why do we need it?

Reply:
These implement NEP-26 callbacks. We're testing our NEP-11 contract, we do not care about callbacks as long as they function the way we need for particular tests.